activesupport/lib/active_support/duration.rb

```diff
@@ -312,6 +319,63 @@ def %(other)
         raise_type_error(other)
```

Comment:
⏫ I think `*(Duration)` is silently treating `other` as Just A Number, so should probably be deprecated too.